### ISSUE TITLE
Route not defined

### ISSUE THREAD

**Issue by Abubaker87** [LaravelDaily/quickadmin#95]
I am using Quick Admin panel. In Admin ItemController, I added a function "MyOwnFunction()". Now when I try to access it in a .blade.php view using link_to_route(config('quickadmin.route').'.item.MyOwnFunction', 'View' , null, array('class' => 'btn btn-success')) !!} I am getting this error.

ErrorException in UrlGenerator.php line 314:
Route [admin.item.MyOwnFunction] not defined. 

Only those routes are working that were created by QuickAdmin CRUD generator. None of my created functions in Controllers are working. 

Please tell me how to define routes or fix this issue. 

Thank you.

**Comment by PovilasKorop** [comment]
Hi @Abubaker87 - for custom Controller functions you need to put them into routes manually. Our QuickAdmin CRUD routes work as resource Controllers, and your custom functions need to be added to routes file separately.
Read more here: [https://laravel.com/docs/5.3/controllers#resource-controllers](https://laravel.com/docs/5.3/controllers#resource-controllers)

**Comment by Abubaker87** [comment]
THank you. I am using Laravel 5.3. I am adding the route like this in my routes > web.php

Route::resource('admin.item.mycustomfunction', 'Admin\ItemController@mycustomfunction');

In a view, I am calling it like this:

<p>{!!  link_to_route('admin.item.mycustomfunction', trans('quickadmin::templates.templates-view_index-add_new') , null, array('class' => 'btn btn-success')) !!}</p>

Still getting the same error. What could I have been missing?

**Comment by PovilasKorop** [comment]
@Abubaker87 to route to a particular function it should be Route::get(), not Route::resource()
Please read the documentation: [https://laravel.com/docs/5.3/routing](url)

**Comment by Abubaker87** [comment]
Sorry to bother you again but I am still getting the same error

Route::get('admin.item.mycustomfunction', 'Admin\ItemController@mycustomfunction');

**Comment by PovilasKorop** [comment]
@Abubaker87 then it could be multiple things - like uppercase/lowercase string, like namespaces etc. Sorry, this one is out of support for our package.

**Comment by Abubaker87** [comment]
Ok. But I would be fine if your package/source code has just one such example so that we could have easily followed that example. Let me try again.

**Comment by PovilasKorop** [comment]
@Abubaker87 our package cannot provide documentation on how people use it AFTERWARDS. Our package generates adminpanel, after that people are free to do whatever they want with their Laravel project, we don't control that.

**Comment by Abubaker87** [comment]
Ok. Thank you. I just wanted to know if there is any other routes file for quickadmin or can we just add our custom routes in routes > web.php
